public/tier1/generichash.h

```diff
@@ -108,7 +108,15 @@ template<> inline unsigned HashItem<char *>(char * const &pszKey )
 uint32 MurmurHash2( const void * key, int len, uint32 seed );
 
 // return murmurhash2 of a downcased string
-uint32 MurmurHash2LowerCase( char const *pString, uint32 nSeed );
+uint32 MurmurHash2LowerCase( char const *pString, int nLength, uint32 nSeed )
+{
+	CBufferStringGrowable<128> buff;
+
+	buff.Insert( 0, pString, nLength );
+	buff.ToLowerFast( 0 );
+
+	return MurmurHash2( buff.Get(), nLength, nSeed );
+}

```

Comment:
Can you move this function body to the generichash.cpp instead? And please replace this part of the code with ``MurMurHash2LowerCase`` then https://github.com/alliedmodders/hl2sdk/blob/3903ac9d50c522969603af1885b83a63ef81ae06/public/igameevents.h#L72-L75

Reply:
Can you also replace the code I linked in the igameevents.h to use ``MurMurHash2LowerCase`` instead